ui/components/app/snaps/snap-ui-renderer/components/container.ts

```diff
@@ -80,6 +80,9 @@ export const container: UIComponentFactory<BoxElement> = ({
       flexDirection: FlexDirection.Column,
       height: BlockSize.Full,
       className: 'snap-ui-renderer__container',
+      style: {
+        overflow: 'auto',
```

Comment:
Why is this now needed?

Reply:
Because the background would only fit the screen as much as the first height of the popup is. It would be cut somewhere when content of custom UI is longer (scrollable).

Reply:
![Screenshot 2024-09-05 at 16 08 52](https://github.com/user-attachments/assets/dd760875-efcf-4339-93f5-2ed01996b66b)
Example of the issue

Reply:
Updated to target only Y axis as well.